MorkoUnityProject/Assets/Scripts/ObservableList.cs

```diff
@@ -1,22 +1,21 @@
 ﻿using System;
 using System.Collections.Generic;
 
+/*
+This class crates a generic list with the functionality of adding, removing at index and returning the size
+When Add() or RemoveAt(), Event is triggered
+*/
+
```

Comment:
Move this at the very top, before any code(including using-statements. Also Add your name here, So we can easily know who to ask in case of questions.